src/manager.rs

```diff
@@ -44,6 +46,24 @@ lazy_static! {
     };
 }
 
+// Workaround the UNC path on Windows, see https://github.com/rust-lang/rust/issues/42869.
+// Otherwise, `Env::from_env()` will panic with error_no(123).
+fn canonicalize_path<'p, P>(path: P) -> io::Result<PathBuf>
+where P: Into<&'p Path> {
+    let canonical = path.into().canonicalize()?;
+    {
```

Comment:
It seems like we should preface this block to compile it only on Windows:

```rust
    #[cfg(windows)]
```

Then other platforms won't pay the cost of the workaround.

Reply:
Good point, will tweak it a bit based on all other review feedback.